frontend/src/components/organisms/MatchesList.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import styled from "styled-components";
+import {Card} from '@material-ui/core';
+import {MatchItem} from "../molecules/MatchItem/MatchItem";
+import {MIN_CARD_COMPONENT_WIDTH} from "../atoms/constants/sizes";
+
+const exampleDataBase: MatchListProps[] = [
+    {
+        onClickTeam: callBackFunction,
+        matchNumber: 0,
+        level: 1,
+        matchStatus: "aktywny",
+        team1: {
+            player1: "Lord Json",
+            player2: "Waszmość Brzuszek",
+            teamNumber: "1",
+            currentPlayerLevel: 0,
+            currentMatchNumber: 0,
+        },
+        team2: {
+            player1: "Kot Długosz Maurycy",
+            player2: "Kot Mańkut Leworęczny",
+            teamNumber: "4",
+            currentPlayerLevel: 0,
+            currentMatchNumber: 0,
+        }
+    },
+    {
+        onClickTeam: () => {
+            console.log("AaaBbbCcc")
+        },
+        matchNumber: 1,
+        level: 1,
+        matchStatus: "aktywny",
+        team1: {
+            player1: "Damian Kołodziej",
+            player2: "Marcel Borkowski",
+            teamNumber: "2",
+            currentPlayerLevel: 0,
+            currentMatchNumber: 0,
+        },
+        team2: {
+            player1: "Bartłomiej Lis",
+            player2: "Jan Wróblewski",
+            teamNumber: "3",
+            currentPlayerLevel: 0,
+            currentMatchNumber: 0,
+        }
+    }
+];
+
+export type MatchListProps = {
+    onClickTeam: () => void,
+    matchNumber: number | undefined,
+    level: number | undefined,
+    matchStatus: string | undefined
```

Comment:
Zdaje mi sie, ze statusy powinnismy miec: 
"oczekuje na rywali"  | "oczekuje na wolny stół" | "do wywołania" | "wywołany" | "zakonczony" 

Co na to @lamparina  :) ?

Reply:
To zmienię w nastepnym PR. Aktualnie nie jestem jeszcze pewien czy i w jaki sposób dostanę to z backendu.
Chciałbym mocno mergować, bo trochę już się rozrosło i przerażają mnie konflikty